mitodl/micromasters
# Reintroduce new course enrollment UX

#### What are the relevant tickets?
Fixes #2467 (again) and fixes #2743.

#### What's this PR do?
Reintroduces the changes from #2519, with a bugfix so that it intelligently picks the correct price from the price map, instead of a random one.

#### How should this be manually tested?
Same procedure as #2519. To test the bugfix, set up two different programs with two different prices, and verify that the correct price is displayed for each program.
